### PR TITLE
Resolver

### PR DESCRIPTION
**Pull request by heffergm**

### CODE REVIEW

File: recipes/proxy.rb [hunk 1]
```diff
@@ -20,6 +20,18 @@
   content '<h1>WORKS!</h1>'
 end
 
+execute 'generate resolver config' do
+  command <<-EOH
+    echo resolver $(awk 'BEGIN{ORS=" "} /nameserver/{print $2}' /etc/resolv.conf | sed "s/ $/;/g") >/etc/nginx/conf.d/resolver.conf
+  EOH
+  notifies :reload, 'service[nginx]'
+end
+
```
Comment (rmarianski) [inline recipes/proxy.rb:29]: Nice :) In other places we've just put hard-coded the google servers, eg 8.8.8.8. Is it worth factoring this logic into a common cookbook to share across others?

File: recipes/proxy.rb [hunk 2]
```diff
@@ -20,6 +20,18 @@
...
+template '/etc/nginx/conf.d/apiaxle-proxy-upstream.conf' do
+  source 'apiaxle-proxy-upstream.conf.erb'
+  notifies :reload, 'service[nginx]'
+end
+
```
Comment (rmarianski) [inline recipes/proxy.rb:34]: Is this section a duplicate of the below? Copy pasta?

Comment (heffergm) [inline recipes/proxy.rb:34]: Yeah, that should be gone.

File: recipes/proxy.rb (hunk 1 above)
Comment (heffergm) [inline recipes/proxy.rb:29]: Maybe... seemed hacky to me, but it works for now ;)
I think a custom ohai fact would probably work better, but I've never had to do them in opsworks and didn't want to spend the time just yet. I'll look into it more.

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**